app/glossary-of-terms.md

```diff
@@ -0,0 +1,100 @@
+---
```

Comment:
Can we rename the file to `glossary.md` too?

Reply:
Done